demos/editdialog/editdialog.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	ui "github.com/Viv1k/clui"
```

Comment:
I know that there is no edit dialog in my repository yet, but I think it is better to use one import for all demos :)

Reply:
Yes, but when I give it your package name then it is not compiling  
I wanted you to change it since currently it is in my fork

Reply:
I agree, I can merge then fix and commit. Anyway, I have one more suggestion for the demo that I've mentioned in conversation as a separate comment

Reply:
Yep. Will make the changes in select box and then send you pull request again... :relaxed: